ui/app/templates/components/dashboard/quick-actions-card.hbs

```diff
@@ -0,0 +1,60 @@
+<h3 class="title is-4">Quick actions</h3>
+
+<div class="has-bottom-margin-m">
+  <h4 class="title is-6">Secrets engines</h4>
+  <SearchSelect
+    @id="secrets-engines-select"
+    @options={{this.secretsEnginesOptions}}
+    @selectLimit="1"
+    @disallowNewItems={{true}}
+    @fallbackComponent="input-search"
+    @onChange={{this.handleSearchEngineSelect}}
+    @placeholder="Type to select a mount..."
+    @displayInherit={{true}}
+    @shouldRenderName={{true}}
+    @passObject={{true}}
+    @objectKeys={{array "type"}}
+    class="is-marginless"
+  />
+</div>
+
+<h4 class="title is-6">Action</h4>
+<Select
+  @name="auth-method"
+  @options={{this.selectedActions}}
+  @valueAttribute="actionType"
+  @labelAttribute="actionTitle"
+  @isFullwidth={{true}}
+  @selectedValue={{this.selectedAction}}
+  @onChange={{this.setSelectedAction}}
+/>
+
+<h4 class="title is-6">{{this.actionParamField.title}}</h4>
+{{#if (eq this.actionParamField.elementType "input")}}
+  <Input @type="text" @value={{this.value}} class="input" />
```

Comment:
Does this update `this.value`? Curious since I don't see an onchange event, but maybe `<Input>` has it built in. 

I think you could pass an on change event directly in the template here removing the need for the conditional in `handleSearch`

 `{{on "change" (mut this.value)}}`

Reply:
I thought I pushed this up, but these Input fields will be removed! :D